### PR TITLE
Stars

### PR DESCRIPTION
**Pull request by Oleksandr-Slieptsov**
- [DEMO LINK](https://Sleptco.github.io/layout_stars/)
- [TEST REPORT LINK](https://Sleptco.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -19,5 +19,11 @@ jobs:
...
-    - run: npm ci
-    - run: npm start & sleep 5 && npm test
+    - run: npm install
+    - run: npm test
+    - name: Upload HTML report(backstop data)
+      if: ${{ always() }}
+      uses: actions/upload-artifact@v2
+      with:
+        name: report
+        path: backstop_data
```
Comment (SerhiiKirik) [inline .github/workflows/test.yml:29]: In the future - Do not add extra files to PR
To add the file to a commit use git add src/.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: 50%;
+  margin-right: 4px;
+}
+
+.stars--5 .stars_star {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
```
Comment (SerhiiKirik) [inline src/style.css:34]: you already have this in class .stars_star
Remove from all cases
```suggestion
```

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Well done 🔥 
Let's improve your code & try to pass the tests
Remove the margin-right from the last star
image

**Comment by Sleptco** [comment]
> Let's improve your code & try to pass the tests



> Well done 🔥 Let's improve your code & try to pass the tests Remove the margin-right from the last star image
image
коли прибираю у останьої зірки марджин, то тест зовсім не проходить.

.stars_star:last-child {
  margin-right: 0;
}

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  height: 16px;
+  background-position: 50%;
+  margin-right: 4px;
+}
+
+.stars_star:last-child {
+  margin-right: 0;
+}
+
+.stars--5 .stars_star {
```
Comment (mykhalenych) [inline src/style.css:26]:
```suggestion
.stars--5 .stars_star,
.stars--1 .stars_star:first-child,
... {
  background-image: url("images/star-active.svg");
  }
```

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 4]
```diff
@@ -8,6 +8,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="frame">
+
+      <div class="stars stars--0">
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html:13]: don't add extra space btw parent and child node

**Review by Volodymyr-Melnyk-Fron10der — APPROVED** [review]
Good, just take a look on one my comment to avoid it in future

**Pull request closed without merging**